tests/hive_integration_test.go

```diff
@@ -1,201 +0,0 @@
-// +build manualintegration
```

Comment:
Do we need to delete this?

Reply:
I don't think so, but they're not working anymore.  I should try to fix them at some point.